instrumentation/java-http-server/library/src/main/java/io/opentelemetry/instrumentation/httpserver/JavaHttpServerAttributesGetter.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.httpserver;
```

Comment:
```suggestion
package io.opentelemetry.instrumentation.javahttpserver;
```

Reply:
not that I mind, but the java 11 HTTP client instrumentation has a similar naming (`httpclient` instead of `javahttpclient`)

Reply:
oh thanks for pointing that out! I added that to fix in #12846